src/components/header/authorizedUser/styles.css

```diff
@@ -1,18 +1,20 @@
 .menu {
   position: relative;
   display: inline-block;
```

Comment:
`display: inline-block;` why do you need it?

Reply:
This is no longer needed. Since I checked something, but it was not needed, and I simply forgot to remove it.